script/Ciel/Day 9/09_01_CIEL09_2B.txt

```diff
@@ -975,58 +1138,65 @@ Why were they designed in such a way that they cannot exist without feeding off
 [sha:e6b3f07805c1900828bc19a97e57ee6fdd0900d5]{
 -- Page 223, Offset 22702. Glued.
 -- 真祖にとって人間は敵にすぎません。
-To the True Ancestors, humanity is nothing more than an enemy.
+ To the True Ancestors, humanity is nothing more than an enemy.
 }
 [sha:8c757b38b4aca765aad4ed5e4241438ef885a303]{
 -- Page 223, Offset 22703.
 -- もともと自然の一部、いえ、この星そのものとリンクしている彼らの能力は、それこそ際限というものがないんです」
-From the outset, they have been linked to a part of nature, no, linked to the planet itself. There's no such thing as a limit on their power."
+Their abilities have always been a part of nature――no, it's more that they are inextricably linked to the planet itself.
+ As such, there is no limit to their power."
 }
 [sha:1cbc21bca665661e64ca304ad7956dd9a16d5b18]{
 -- Page 224, Offset 22704.
 -- 「教会の歴史の中でも真祖と戦った記録は数えるほどしかありません。
-"Even in the history of the Church, there are but a handful of encounters with True Ancestors that have been recorded.
+"Throughout the entire history of the Church,
+ there have been only a handful of documented encounters with the True Ancestors.
 }
 [sha:80c569a007266272b4361a20275f6a6db47e8f91]{
 -- Page 224, Offset 22705.
 -- <真祖|かれら>は世界そのものから力を引き上げる。<真祖|かれら>を倒すということは世界を倒すほどの概念武装が必要になります。
-True Ancestors derive their power from the world itself. To defeat them would require a Conceptual Weapon on a scale tantamount to defeating the world.
+True Ancestors derive their power from the planet itself.
+ To defeat them would require a Conceptual Weapon powerful enough to bring about the end of the world.
 }
 [sha:3006f129c4eedb536c41138f92303980d714be29]{
 -- Page 224, Offset 22706.
 -- ……もちろん、そんな武装はありません。
-...Obviously, such a weapon does not exist.
+Obviously, such a weapon does not exist.
 }
 [sha:a8ff09349ccbaabd9cc5796af905faf2796bf35a]{
 -- Page 224, Offset 22707.
 -- ですから<真祖|かれら>には、外的要因による“死”を与える事ができない」
-In other words, it is impossible to inflict 'death' upon them through external means."
+// God I hate 外的要因
+As such, it is impossible for any external factors to bring about their 'death'."
 }
 [sha:757ce28f7052e1f44c63fe04823fab3e18af8dfe]{
 -- Page 225, Offset 22708.
 -- 「……さらにはこんな碑文も残されています。
-"...Finally, the following inscription has been preserved in our records:
+"Furthermore, the following inscription has been preserved in our records:
 }
 [sha:f7f89553f929e2d253652dc2374fc38183496e02]{
 -- Page 225, Offset 22709.
 -- “真祖を討つのであれば万全のまま討ってはならない。
-'To vanquish a True Ancestor is to vanquish them while enfeebled.
+'If you would strike at a True Ancestor, strike not when they are in their prime.
```

Comment:
I remember translating this in the past, looking it up...

'Whence one is to strike the True Ancestor,
one mustn't strike whence they be in their fullness.
One ought to vanquish them whence they be on death's door.'

Well, I guess I was having fun. Thought comments and love appreciated.

Reply:
It does hit me a little heavy with the whencing. I'm hesitant to use þe olde speech more than necessary.

Reply:
Yeah, just felt like sharing, apparently.
That said, there is something to be said about it being some kind of saying.